README.adoc

```diff
@@ -0,0 +1,16 @@
+:repository-owner: arduino-libraries
+:repository-name: Arduino_PF1550
+
+= {repository-name} Library for Arduino =
+
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml/badge.svg["Check Arduino status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]

```

Comment:
The GitHub Actions workflows these badges are associated with weren't added to the repository. 

The workflows definitely should be added, but the badges should not be added until the workflows are in place, otherwise the broken badge images are just confusing and look bad.

If you would like to add the workflows, the instructions for installation are here:

- https://github.com/per1234/.github/blob/main/workflow-templates/check-arduino.md
- https://github.com/per1234/.github/blob/main/workflow-templates/compile-examples.md
- https://github.com/per1234/.github/blob/main/workflow-templates/spell-check.md

Reply:
Hello @per1234 , 
I went through the documentation. Unfortunately, I cannot understand how to install the GitHub actions from the provided links. Could you add the actions, or alternatively I can remove lines 6-8?

Reply:
> Could you add the actions

OK, I have done it. Please review: https://github.com/arduino-libraries/Arduino_PF1550/pull/4